### PR TITLE
Tegy-Quote

### PR DESCRIPTION
**Pull request by SondraE**
red-panda

## Description
Added user quote that Karen sent to the TimeDesigner work page. Got permission from Amy to add her quote and picture to OddSite.

## Steps to test/reproduce
/work/timedesigner/

### CODE REVIEW

**Comment by SondraE** [comment]
[Add customer quote to Tegy.](https://trello.com/c/4OggA8av/136-add-customer-quote-to-tegy)

File: report.20210618.141351.42711.0.001.json [hunk 1]
```diff
@@ -0,0 +1,635 @@
...
+    "dumpEventTimeStamp": "1624047231366",
+    "processId": 42711,
+    "cwd": "/Users/sondrae/Sites/oddleventy",
+    "commandLine": [
+      "/Users/sondrae/.nvm/versions/node/v12.16.1/bin/node",
+      "/Users/sondrae/Sites/oddleventy/.yarn/cache/chokidar-cli-npm-2.1.0-3bc4fdea14-6d174178b4.zip/node_modules/chokidar-cli/index.js",
+      "**/.stylelintrc.yml",
+      "-c",
+      "yarn lint:sass"
+    ],
```
Comment (stacyk) [inline report.20210618.141351.42711.0.001.json:18]: I'm guessing we would want to delete this file? and maybe add it to gitignore? Can you confirm @jgerigmeyer

**Review by stacyk — APPROVED** [review]
Looks good now :)

**Review by mirisuzanne — APPROVED** [review]
looks good!

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**